hw_diag/tests/diagnostics/test_version_endpoint.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+from unittest.mock import patch, mock_open
+from hm_pyhelper.diagnostics.diagnostics_report import \
+    DIAGNOSTICS_PASSED_KEY, DIAGNOSTICS_ERRORS_KEY, DiagnosticsReport
+
+from hw_diag.diagnostics.mac_diagnostics import MacDiagnostic, MacDiagnostics
+
+
+class TestMacDiagnostics(unittest.TestCase):
```

Comment:
@kashifpk this looks like you copied the test and never updated it for the version endpoint.

Reply:
@marvinmarnold - Sorry that's clumsy of me. The actual test was in `hw_diag/tests/test_views_diagnostics.py`. I had copied this file earlier to add test and later figured that `test_views_diagnostics.py` was the place to put the test in and this file accidentally got left in the commit.